test/src/server/routes/entity/entity.js

```diff
@@ -80,3 +84,436 @@ describe('getDefaultAliasIndex', () => {
 		expect(getDefaultAliasIndex(aliasSet)).to.equal(2);
 	});
 });
+
+describe('deleteRelationships', () => {
+	const {Author, Work, Revision, WorkRevision, WorkHeader, Relationship, RelationshipSet, RelationshipType, bookshelf} = orm;
+	let author;
+	let editorJSON;
+
+	beforeEach(async () => {
+		const editor = await createEditor();
```

Comment:
The editor you can create once and reuse (you'll need to move it to a `before`  function).

For that to work, you'll need to change the `afterEach` below to an `after`.

I think this should be fine since you overwrite the `author` variable  before each test.